app/frontend/components/MealsList.jsx

```diff
@@ -0,0 +1,40 @@
+"use client";
```

Comment:
You don't need to use `'use client'` here since you're using Vite instead of Next.js

Reply:
👍